### ISSUE TITLE
StdLibAdapter can not extract log line with set prefix correctly

### ISSUE THREAD

**Issue by soven** [go-kit/kit#1035]
When using a prefix in stdlib's logger StdLibAdapter extracts the fields from log line incorrectly. 

For example, the program
```go
package main

import (
    "log"
    "os"

    gokitlog "github.com/go-kit/kit/log"
)

func main() {
    logg := log.New(gokitlog.NewStdlibAdapter(gokitlog.NewLogfmtLogger(os.Stdout)),
        "some prefix ", log.LstdFlags)
    logg.Println("log line")
}
```
will give us the following to the stdout:
```
caller="some prefix 2020/11/16 22:33" msg=":04 log line"
```

I have a fix to support the prefix mode by adding a method `Prefix(prefix string, joinPrefixToMsg bool) StdlibAdapterOption`

**Comment by ChrisHines** [comment]
We would be happy to review a PR to fix this issue.

**Comment by soven** [comment]
done

**Comment by sagikazarmark** [comment]
Fixed in #1036
